source/documentation/requirements.html.md

```diff
@@ -0,0 +1,142 @@
+---
+author: amoralej
+title: General purpose requirements
+---
+
+# Requirements management in RDO 
+
+1. toc
+{:toc}
+
+## Introduction
+
+OpenStack services usually need some pieces of software which are not developed as
+part of the project. They are are general purpose libraries (typically python
+modules) or services used in some way to run or build OpenStack packages.
+
+OpenStack [requirements project](https://docs.openstack.org/developer/requirements/) 
+defines the policies and processes to manage requirements in upstream projects from
+a global perspective.
+
+## Managing OpenStack requirements in RDO
+
+RDO provides all requirements for packaged services in rpm format from their own repos,
+so that no software should be installed from external repositories. This packages can
+be provided by:
+
+- CentOS base repositories (base, updates and extras). This is the preferred source of
+packages whenever possible.
+- Other [CentOS SIG repositories](https://wiki.centos.org/SpecialInterestGroup) (Virtualization,
+Storage, etc...). When a required package is being maintained by other CentOS SIG, it
+will be reused for RDO repos.
+- RDO CloudSIG repositories. When a package is not available from previous repos, it will
+be provided by RDO repositores. Note that it's required that these packages exist previously
+in Fedora so that they can be rebuilt with minimal changes (if any).
+
+If you have questions or special requests, don't hesitate in contacting RDO using our
+[mailing lists](/community/mailing-lists/) or #rdo channel in freenode.
+
+### Adding a new requirement to RDO
+
+When a new requirement is needed for a OpenStack project included in RDO, following process
+must be followed by package maintainers:
+
+![RDO dependencies](/images/cbs-requirements.png)
+
+
+1. If the project follows global-requirements processes, make sure that the requirement has been
+added to global-requirements.txt and upper-constraints.txt files as described in the [upstream
+documentation](https://github.com/openstack/requirements/#proposing-changes)
+    
+    <br />
+2. Check if the new requirement is present in CentOS base channels. The easies way to do this
+is using yum command from a system running CentOS 7:
+    
+        yum list "*<dependency>"
```

Comment:
that'd be nice but I couldn't figure out something better for centos. Additionally, there is no web based package database (or at least i couldn't find it) that i can link.